docs/language/learn-ql/ql-training.rst

```diff
@@ -1,19 +1,19 @@
-QL training and variant analysis examples
-#########################################
+CodeQL training and variant analysis examples
+=============================================
 
 QL and variant analysis
-=======================
+-----------------------
 
 `Variant analysis <https://semmle.com/variant-analysis>`__ is the process of using a known vulnerability as a seed to find similar problems in your code. Security engineers typically perform variant analysis to identify possible vulnerabilities and to ensure that these threats are properly fixed across multiple code bases.
 
-`QL <https://semmle.com/ql>`__ is Semmle's variant analysis engine, and it is also the technology that underpins LGTM, Semmle's community driven security analysis platform. Together, QL and LGTM provide continuous monitoring and scalable variant analysis for your projects, even if you don’t have your own team of dedicated security engineers. You can read more about using QL and LGTM in variant analysis in the `Semmle blog <https://blog.semmle.com/tags/variant-analysis>`__.
+`CodeQL <https://semmle.com/ql>`__ is the code analysis engine that underpins LGTM, Semmle's community driven security analysis platform. Together, CodeQL and LGTM provide continuous monitoring and scalable variant analysis for your projects, even if you don’t have your own team of dedicated security engineers. You can read more about using CodeQL and LGTM in variant analysis in the `Semmle blog <https://blog.semmle.com/tags/variant-analysis>`__.
 
-The QL language is easy to learn, and exploring code using QL is the most efficient way to perform variant analysis. 
+CodeQL is easy to learn, and exploring code using CodeQL is the most efficient way to perform variant analysis. 
 
 Learning QL for variant analysis
```

Comment:
```suggestion
Learning CodeQL for variant analysis
```

Reply:
It seems I didn't push the commit that updated these titles 🤦‍♂ 
Sorry for creating the extra reviewing work!